src/core/context/MainWrapper.tsx

```diff
@@ -22,26 +24,22 @@ const MainWrapper: FC<{ children: ReactNode }> = ({ children }) => {
     setFunctionalCheckbox,
   } = useCookiesContextTracking();
 
-  useLayoutEffect(() => {
-    if (isShowBanner) {
-      setTimeout(() => {
-        window.scrollTo(0, 0);
-        const pageWrapper = getPageWrapper();
-        if (pageWrapper) {
-          pageWrapper.style.overflow = 'hidden';
-        }
-      }, 100);
-      lockScroll();
+  useEffect(() => {
+    if (isShowBanner && router.pathname !== '/cookies-policy') {
```

Comment:
We should check this: `router.pathname.startsWith(siteRoutes.cookiesPolicy)`. Use the `startsWith` method because it could contains hash and they're added to the pathname in clientside, so it can raise hydratation errors as hashes are not present in server side or return false as the pathname is no exactly what we expect